docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -206,103 +208,151 @@ mode uses block devices to create the thin pool. The following procedure shows
-          [1] 2163
-          [root@ip-10-0-0-75 centos]# INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
-          INFO[0027] Option DefaultDriver: bridge
-          INFO[0027] Option DefaultNetwork: bridge
-          <output truncated>
-          INFO[0027] Daemon has completed initialization
-          INFO[0027] Docker daemon commit=1b09a95-unsupported graphdriver=aufs version=1.11.0-dev
+    The value's setting is the perentage of space to increase the thin pool (100 =
+    disabled)
 
-    It is also possible to set the `--storage-driver` and `--storage-opt` flags
- in the Docker config file and start the daemon normally using the `service` or
- `systemd` commands.
+    ```
+    thin_pool_autoextend_percent = 20
+    ```
+    
+11. Check your work, your `docker-thinpool.profile` file should appear similar to the following:
 
-8. Use the `docker info` command to verify that the daemon is using `data` and
-`metadata` devices you created.
+    ```
+     example /etc/lvm/profile/docker-thinpool.profile:
+     activation {
+         thin_pool_autoextend_threshold=80
+         thin_pool_autoextend_percent=20
+     }
+     ```
 
-        $ sudo docker info
-        INFO[0180] GET /v1.20/info
-        Containers: 0
-        Images: 0
-        Storage Driver: devicemapper
-         Pool Name: docker-202:1-1032-pool
-         Pool Blocksize: 65.54 kB
-         Backing Filesystem: xfs
-         Data file: /dev/vg-docker/data
-         Metadata file: /dev/vg-docker/metadata
-        [...]
+12. Apply your new lvm profile
+
+    ```bash
+    $ lvchange --metadataprofile docker-thinpool docker/thinpool
+  ```
+
+13. Verify the `lv` is monitored.
+
+    ```bash
+    $ lvs -o+seg_monitor
+    ```
+
+14. If Engine was previously started, clear your graph driver directory.
+
+    ```bash
+    $ rm -rf /var/lib/docker/*
+    ```
+
+14. Configure the Engine daemon with specific devicemapper options.
+
+    There are two ways to do this. You can set options on the commmand line if you start the daemon there:
+
+    ```bash
+    --storage-driver=devicemapper --storage-opt=dm.thinpooldev=/dev/mapper/docker-thinpool --storage-opt dm.use_deferred_removal=true
+    ```
+
+    You can also set them for startup in the `daemon.json` configuration, for example:
+
+    ```json
+     {
+             "storage-driver": "devicemapper",
+             "storage-opts": [
+                     "dm.thinpooldev=/dev/mapper/docker-thinpool",
+                     "dm.use_deferred_removal=true"
+             ]
+     }
+    ```
+15. Start the Engine daemon.
+
+    ```bash
+    $ systemctl start docker
+    ```
+
+After you start the Engine daemon, ensure you monitor your thin pool and volume
+group free space. While the volume group will auto-extend, it can still fill
+up. To monitor logical volumes, use `lvs` alone or `lvs -a` alone to see tha
```

Comment:
In a procedure, I don't like to link out because it can be distracting. So, I have a link to the options at the end of this \ section (353) and in the Related information.